ibc/test_relay.go

```diff
@@ -7,174 +7,6 @@ import (
 	transfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
 )
 
-func (ibc IBCTestCase) RelayPacketTest(testName string, cf ChainFactory, relayerImplementation RelayerImplementation) error {
```

Comment:
This function was effectively moved to `relayertest.TestRelayer_RelayPacket`.

Reply:
Why was only this function moved?